src/main/java/mate/academy/bookstore/config/SecurityConfig.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.bookstore.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Configuration
+public class SecurityConfig {
+    @Bean
+    public BCryptPasswordEncoder getPasswordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
```

Comment:
do we need securityFilterChain here?

Reply:
![Untitled](https://github.com/user-attachments/assets/208cfc46-0aab-4078-8eb8-4a06a7f42b78)